Migrate to peewee moves

AMecea opened the pull request:
- [x] needs documentation update

ccrisan commented on colibris/conf/settings.py:
```diff
@@ -52,8 +52,8 @@
-        'peewee_migrate': {
-            'level': 'DEBUG'  # Always show details when doing migrations
+        'peewee_moves': {
+            'level': 'DEBUG'   # Always show details when doing migrations
```
Why more than 2 spaces?

ccrisan commented:
What's the purpose of the added `__packagename__.db` file?

AMecea commented:
Thanks, @ccrisan for the review. I've updated the code accordingly. Also, I've added documentation of how to use Peewee Moves and how to upgrade from Colibris v0.9 to v0.10.

AMecea commented:
Hi @ccrisan, Now, peewee-moves have fake migrations support. This PR is ready for review now. Thanks!

ccrisan commented on docs/source/Migrations.md:
```diff
@@ -1,17 +1,137 @@
 
+
+## Manage Migrations
+The migrations are managed by [`peewee-moves`](https://github.com/timster/peewee-moves) a small and
```
I recommend a `,` comma before `a small...`

ccrisan commented on docs/source/Migrations.md:
```diff
@@ -1,17 +1,137 @@
+
+## Upgrade model
+
+Peewee Moves support only creating a new model. The alteration of an existing model is done manually
```
`supports`

ccrisan commented on docs/source/Migrations.md:
```diff
@@ -1,17 +1,137 @@
+
+    ./${PACKAGE}/manage.py db revisions alter_model
+    
+Then update the newly generated file `migrations/0XXX_alter_model.py` to contains the model changes.
```
`to contain`

ccrisan commented on docs/source/Migrations.md:
```diff
@@ -1,17 +1,137 @@
+## Upgrade model
+
+Peewee Moves support only creating a new model. The alteration of an existing model is done manually
+by generating a new revision and editing it. For generating an empty migration use:
```
`, use`

ccrisan commented on docs/source/Migrations.md:
```diff
@@ -1,17 +1,137 @@
+
+### Adding/Removing/Renaming a model field
+
+Create an empty revision file with the above command `migrations/0004_alter_user_fields.py` then
```
Please add some commas to make this sentence more intelligible.

ccrisan commented on docs/source/Migrations.md:
```diff
@@ -1,17 +1,137 @@
+    migrator.rename_column('user', 'key', 'password')
+
+def downgrade(migrator):
+    # Remove dispaly name field
```
`display_name`

ccrisan commented on docs/source/Migrations.md:
```diff
@@ -1,17 +1,137 @@
+    # Add role field
+    migrator.add_column('user', 'role', 'char', max_length=10)
+    
+    # Rename
```
`Rename password field to key.

ccrisan commented on docs/source/Migrations.md:
```diff
@@ -1,17 +1,137 @@
+
+### Python script
+
+Peewee Migrate allows running SQL queries directly in the migrations. Here is an example of how to
```
Isn't this a copy-paste leftover?

ccrisan commented on docs/source/Migrations.md:
```diff
@@ -1,17 +1,137 @@
+
+## Upgrade from peewee-migrate
+
+Colibris version `<=0.9` used for migrations `peewee-migrate` but starting with version `0.10` it
```
I would add some commas and reorganize sentence topic, e.g.:

    Colibris version `<=0.9` used `peewee-migrate` for migrations, but starting with version `0.10`, it uses `peewee-moves`, a more lightweight manager.

ccrisan commented:
@AMecea aside from a few grammar mistakes, LGTM.
